contracts/Org.sol

```diff
@@ -54,40 +58,67 @@ contract Org is Administratable {
 
   /**
    * @notice Create Organization Claim
+   * @param  claimId UUID representing this claim
    * @param  fName First name of Administrator
    * @param  lName Last name of Administrator
    * @param  eMail Email contact for Organization Administrator.
    * @param  orgAdminWalletAddress Wallet address of Organization's Administrator.
    */
   function claimRequest(
+    string memory claimId,
```

Comment:
When rebased against the master, the rest of the inputs will have validation to prevent empty strings. I suggest we rebase against the master now, then add that check on `claimId` here as well.

This has the side effect of preventing the need from checking for an empty `claimId` in `rejectClaim`

Reply:
Sounds good.

Reply:
Did this rebase and update happen @mds1 ? Just looking to close out convos

Reply:
It will be happening today!